src/index.ts

```diff
@@ -213,19 +214,65 @@ server.tool(
   },
 );
 
+server.tool(
+  "get_follower_ids",
+  "Gets the list of User IDs of users who have added your LINE Official Account as a friend.",
+  {
+    start: z
+      .string()
+      .optional()
+      .describe(
+        "Continuation token for pagination. If omitted, fetches from the beginning.",
```

Comment:
```suggestion
        "Value of the continuation token found in the next property of the JSON object returned in the response. Include this parameter to get the next array of user IDs. If omitted, fetches from the beginning.",
```
How about describing about using next property like https://developers.line.biz/en/reference/messaging-api/#get-follower-ids-query-parameters

Reply:
Please fix this 🙏